wfe/wfe.go

```diff
@@ -983,8 +984,11 @@ func (wfe *WebFrontEndImpl) Challenge(
 
 	authz, err := wfe.SA.GetAuthorization(ctx, authorizationID)
 	if err != nil {
-		// TODO(#1198): handle db errors etc
-		notFound()
+		if err == sql.ErrNoRows {
+			notFound()
+		} else {
+			wfe.sendError(response, logEvent, probs.ServerInternal("Internal Server Error"), err)
```

Comment:
Leaving the detail blank is also a departure from common practice in WFE. How about "problem getting authorization"?